tests/lib/gladecheck.py

```diff
@@ -38,6 +38,8 @@
 import os
 import unittest
 import copy
+# FIXME: Enable glade tests back when we found a way how to migrate them to python3 unittest
+# pylint: disable=import-error
```

Comment:
We could probably convert this to "just" a normal test that runs standalone, and use [`subTest`](https://docs.python.org/3/library/unittest.html?highlight=unittest#distinguishing-test-iterations-using-subtests) to report the failures in a sane way.

Reply:
Seems like a good idea but not for this PR.